frontend/src/pages/ui/FeedbackBox.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+// @ts-ignore
+import Feedback from "@benhammondmusic/feeder-react-feedback";
+import "@benhammondmusic/feeder-react-feedback/dist/feeder-react-feedback.css"; // import stylesheet
+
+import { useBottomScrollListener } from "react-bottom-scroll-listener";
+import { useCookies } from "react-cookie";
+
+const FEEDBACK_ID = "6171cc2965b82c00045239dc"; // view collected feedback at feeder.sh/dashboard
```

Comment:
I think we should pass this in as secret.

Reply:
https://github.com/SatcherInstitute/health-equity-tracker/blob/main/.github/workflows/deployToTest.yml#L86

Reply:
Currently I set it up to read from process.env or this ID. once we have production working reading github secrets, we can create a new dashboard account at feeder.sh and use that new secret id